src/types/Account/methods/getUTXOS.js

```diff
@@ -7,15 +8,23 @@ function getUTXOS() {
   const utxos = [];
 
   const self = this;
-  const { walletId, network } = this;
+  const {
+    walletId, network, BIP44PATH, walletType,
+  } = this;
   const currentBlockHeight = this.store.chains[network].blockHeight;
   /* eslint-disable-next-line no-restricted-syntax */
-  for (const walletType in this.store.wallets[walletId].addresses) {
-    if (walletType && ['external', 'internal', 'misc'].includes(walletType)) {
+  for (const addressType in this.store.wallets[walletId].addresses) {
+    if (addressType && ['external', 'internal', 'misc'].includes(addressType)) {
```

Comment:
Super nested, hard to read.

Maybe try to decrease levels with:

```js
if (!addressType || !['external', 'internal', 'misc'].includes(addressType)) {
  continue;
}
```

http://blog.wilsonpage.co.uk/return-early/
https://blog.timoxley.com/post/47041269194/avoid-else-return-early